src/transport_ofi.h

```diff
@@ -14,6 +14,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <sys/uio.h>
```

Comment:
By the way, it's slightly surprising this is needed.  It's not in `fi_rma.h`?

Reply:
This will be fixed upstream in https://github.com/ofiwg/libfabric/pull/4541, but we'll need to keep this for compatibility with current and previous releases of libfabric.